scripts/mkcloud

```diff
@@ -15,6 +15,15 @@
 # 2. This tool relies on the script qa_crowbarsetup.sh
 # 3. Please 'export' environment variables according to your needs.
 
+if [[ "$cloud" ]]; then
```

Comment:
I agree. There is no point in testing for a variable, that gets a default value some lines below.